ci/release/update-version.sh

```diff
@@ -55,3 +55,7 @@ for FILE in .github/workflows/*.yaml; do
   sed_runner "/shared-action-workflows/ s/@.*/@branch-${NEXT_SHORT_TAG}/g" "${FILE}"
 done
 sed_runner "s/RAPIDS_VERSION_NUMBER=\".*/RAPIDS_VERSION_NUMBER=\"${NEXT_SHORT_TAG}\"/g" ci/build_docs.sh
+
+# .devcontainer files
+sed_runner "s/ARG RAPIDS=${CURRENT_SHORT_TAG}/ARG RAPIDS=${NEXT_SHORT_TAG}/g" .devcontainer/Dockerfile
+find .devcontainer/ -type f -name devcontainer.json -exec sed -i "s/${CURRENT_SHORT_TAG}/${NEXT_SHORT_TAG}/g" {} \;
```

Comment:
A few comments here:

- Is the `.devcontainer/Dockerfile` `sed` command necessary? I don't see any RAPIDS versions in that Dockerfile. Not sure if this was refactored out at some point
- For the reasons outlined in [this comment](https://github.com/rapidsai/ucxx/pull/36#discussion_r1178429239), I've been discouraging the use of variables (e.g. `CURRENT_SHORT_TAG`) in the search pattern of these `sed` commands since it makes the commands fragile. Additionally, these `update-version.sh` scripts will likely be refactored in the future to be passed a single argument, `RAPIDS_VERSION`, instead of distinguishing between "current" and "next" versions.
- Historically, we've intentionally used `sed_runner` instead of `sed` since it works Mac and Linux machines. This is important because the `update-version.sh` currently gets run locally by the Ops team @ release time and therefore needs to work on both machine types. We have some plans to revamp how these updates are handled in the future, but for now, it's probably best to stick to the status quo to prevent any unforeseen issues.

Reply:
> Is the .devcontainer/Dockerfile sed command necessary?

It was at one point, but the RAPIDS version was moved into each devcontainer.json file.

> Additionally, these update-version.sh scripts will likely be refactored in the future to be passed a single argument, `RAPIDS_VERSION`, instead of distinguishing between "current" and "next" versions.

That's fine, I can update the script to match explicit strings.

> Historically, we've intentionally used sed_runner instead of sed since it works Mac and Linux machines.

Ok, I'll make the `find` exec line do what the `sed_runner` does. I think the only alternative would be exporting `sed_runner` to the current env, and that sounds less desirable.